SPECS/grub2/grub2.spec

```diff
@@ -291,7 +293,7 @@ cp $GRUB_PXE_MODULE_SOURCE $EFI_BOOT_DIR/$GRUB_PXE_MODULE_NAME
 %{_bindir}/*
 %{_datarootdir}/grub/*
 %{_sysconfdir}/sysconfig/grub
-%{_sysconfdir}/default/grub
+%attr(0644,root,root) %ghost %config(noreplace) %{_sysconfdir}/default/grub
```

Comment:
grub package should own/provide the default/grub.d dir also right?

Reply:
Hmm, definitely need to package that %dir.

Reply:
don't think this is done yet is it?

Reply:
I think the directory the config files are placed in should be independent of the main grub package? i.e., we should be able to install the config files even if we don't have grub installed. Maybe see what the existing setup looks like for systemd?